pages/about.html

```diff
@@ -1,12 +1,34 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="about-D">
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../style.css">
   <title>Document</title>
 </head>
 <body>
+  <header>
+ <hi class="about-header">Hola!</hi>
+</header>
+<main>
+  <img class="about-img"  src="/images/Screenshot (21).png" alt="Diana">
+    <ul>
+        <p class="Dianas-paragraph">My name is Diana, I am a current student at ADA, on my way to become a full stack programmer</p>
+        <p class="Dianas-paragraph">I love to create things, I love my dogs, I love to travel </p>
+        <p class="Dianas-paragraph">Tools/languages I can use:
+            <li class="Dianas-paragraph">Python</li>
+            <li class="Dianas-paragraph">Flask</li>
+            <li class="Dianas-paragraph">Postman</li>
+            <li class="Dianas-paragraph">SQL</li>
+            <li class="Dianas-paragraph">Heroku</li>
+            <li class="Dianas-paragraph">Github</li>
+        <p class="Dianas-paragraph">Feel free to explore my previous projects:</p>
+        <a class ="Dianas-paragraph" href="portfolio.html">My creations</a>
+
+        </p>
+    </ul>
```

Comment:
These should be after the list
```suggestion
    </ul>
    <p class="Dianas-paragraph">Feel free to explore my previous projects:</p>
    <a class ="Dianas-paragraph" href="portfolio.html">My creations</a>
    </p>
```